WPMedia/fusion-news-theme-blocks
# ASUB-8549 custom buttom for facebook and apple - updating user accoun…

…t management

**Be sure to run `npm test`, `npm run lint`, and write detailed test steps before requesting reviewers**

## Description
Updates on Social-sign-on & account management block

https://arcpublishing.atlassian.net/browse/ASUB-8521
https://arcpublishing.atlassian.net/browse/ASUB-8549

There are some AppSec Security alerts, All these will be fixed as part of https://arcpublishing.atlassian.net/browse/ASUB-8539

## Test Steps

Use the cdn endpoint https://stagingva-staging-sandbox.api.cdn.arcpublishing.com
Since this tenant has all the Third Auth Providers.

2 Blocks has been updated as part of this PR
Create a page with a Social-Sign-on block. Enable/disable the custom field **customButtons**. It will render the buttons provided by Fb or Google or the custom buttons included as part of this ticket. 
Create a page with account-management block. There you should be able to see the identities attached, or you should be able to attach new identities. 

1. Checkout this branch `git checkout ASUB-8521-SocialMediaButtons`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/identity-block`
3. Create a page with a Social-Sign-on block. Enable/disable the custom field **customButtons**. It will render the buttons provided by Fb or Google or the custom buttons included as part of this ticket. 
4 Create a page with account-management block. There you should be able to see the identities attached, or you should be able to attach/remove identities.
